src/validators/bankruptcy.js

```diff
@@ -31,7 +31,9 @@ export default class BankruptcyValidator {
    * Validates the yes/no branching for bankruptcy
    */
   validHasBankruptcy() {
-    return validateModel(this.data, { HasBankruptcy: financialBankruptcyModel.HasBankruptcy }) === true
+    return validateModel(this.data, {
```

Comment:
do we have something like prettier installed to automagically do this & enforce a standard?

Reply:
we do technically have a prettier config, although I think it currently conflicts with some of our eslint rules. Right now eslint happens to staged files as a pre commit hook, that's what we've been using so far. I agree it would be great to just have it taken care of with prettier but I think that's outside the scope of this PR :)